catalogs/mississippi-s1/request_builders.py

```diff
@@ -0,0 +1,184 @@
+import datetime
```

Comment:
Shoot. `request_builders.py` and `s3_io.py` are copied without modification from other catalog builders, right? 

I attempted to move these up a level to `./catalogs` over in #90 since we now have a few subprojects using these but when I did I ran into trouble with relative imports throwing `Attempted relative import with no known parent package` even though I added an `__init__.py` to `./catalogs` and all intermediate dirs. In that PR, I worked around the issue by explicitly passing PYTHONPATH, but that's not a sustainable solution because you have to remember to pass it every time you run one a script.

Reply:
Opened https://github.com/azavea/noaa-flood-mapping/issues/94